app/services/update_wikidata_stats.rb

```diff
@@ -1,32 +1,132 @@
 # frozen_string_literal: true
-
 require_dependency "#{Rails.root}/lib/wikidata_summary_parser"
 require_dependency "#{Rails.root}/lib/importers/wikidata_summary_importer"
+# require the installed wikidata-diff-analyzer gem
+require 'wikidata-diff-analyzer'
 
 class UpdateWikidataStats
+  STATS_CLASSIFICATION = [
+    'claims created', 'claims removed', 'claims changed',
+    'references added', 'references removed', 'references changed',
+    'qualifiers added', 'qualifiers removed', 'qualifiers changed',
+    'aliases added', 'aliases removed', 'aliases changed',
+    'labels added', 'labels removed', 'labels changed',
+    'descriptions added', 'descriptions removed', 'descriptions changed',
+    'interwiki links added', 'interwiki links removed', 'interwiki links updated',
+    'merged to', 'merged from', 'redirects created',
+    'reverts performed', 'restorations performed', 'items cleared',
+    'items created', 'lemmas added', 'lemmas removed', 'lemmas changed',
+    'forms added', 'forms removed', 'forms changed',
+    'senses added', 'senses removed', 'senses changed',
+    'properties created', 'lexeme items created',
+    'representations added', 'representations removed', 'representations changed',
+    'glosses added', 'glosses removed', 'glosses changed',
+    'form claims added', 'form claims removed', 'form claims changed',
+    'sense claims added', 'sense claims removed', 'sense claims changed'
+  ].freeze
+
   def initialize(course)
     @course = course
-    import_summaries
+    # import_summaries
+    update_summary_with_stats
     update_wikidata_statistics
   end
 
   private
 
+  # When summary is nil, instead of fetching edit summaries for each revision, I want to use
```

Comment:
Comments should not be written in first-person, unless you're signing them to explicitly indicate who "I" is. (In contrast, I think first-person comments are fine in commit descriptions, as those are explicit signed and it's useful to understand the personal perspective of the committer.)